assets/js/src/delete-posts-by-comments.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * JavaScript for delete posts by comments module of Bulk Delete Plugin.
+ *
+ * @author Sudar <https://bulkwp.com>
+ * @since 6.1.0
```

Comment:
Let's change the version to `6.0.0`. I am planning to release it as part of `6.0.0` itself.

Reply:
Fixed.